product_code_uniqueness/models/product_product.py

```diff
@@ -46,7 +51,7 @@ def _check_default_code(self):
                 )
                 if matches > 1:
                     raise ValidationError(
-                        _(
```

Comment:
there is no longer `_` importable translation?

Reply:
Nice, thats actually better approach. Previously there were cases when translations would not work and it would silently ignore it, for this same reason as it could not identify `env` (e.g. using translations in functions without `env`).